command_list.md

```diff
@@ -0,0 +1,132 @@
+# Account
+```
+account [list]
+account add 
+account info 
+account remove 
+account usage 
+account set
+
+account ban add
+account ban remove
+
+account attribute list 
+account attribute add 
+account attribute remove 
+
+account limit list 
+account limit add 
+account limit remove
+
+account identity list
+account identity add 
+account identity remove 
+```
+
+# Config 
+```
+config [list]
+config set
+config remove
+```
+
+# DID 
+```
+did [list]
+did stat
+did remove
+did touch
+did parent
+did attach
+did detach
+did upload
+did download
+
+did content list
+did content history
+did content close
+did content open
+
+did metadata list
+did metadata add
+did metadata remove
+
+did container add
```

Comment:
Let me get ahead of questions: These two are on their own because it would be easier to do that now and add additional operations than do an additional refactor down the road if this functionality is expanded on

Reply:
Forgive me, GitHub is making this comment ambiguous. Are you referring to `metadata` or `container`?

Reply:
Ah, this was referring to the container/dataset add

Reply:
Thanks for the clarification. Let me copy my comment from below:

> I don’t think it’s worth having the container and dataset groups. Both of them are collections, and you can’t add a file DID. I would recommend putting the type of collections as an option to `did add`.

So, could you expand on 1) why it would be easier to do that now and 2) what possible future expansion did you have in mind?

Reply:
1. It'll just have the structure in place now rather than later
2. I don't know the usefulness of it (after our other conversations), but it would be easier to separate it out into `did container[dataset] list/add/remove`. More explicit than using kwargs that may or may not have defaults that make things unnecessarily confusing.

Reply:
I’m afraid I do not follow you reasoning.

1. There is no reason to highlight the differences between dataset and containers. Both are collections. At the Rucio Workshop, there was a discussion about whether in the future the should be unified (i.e. drop the restriction that containers can’t contain files, or that datasets can’t contain collections).
2. It’s more confusing to not follow the `entity {list,show,add,update,delete}` model. What you’re suggesting will require two groups, with either missing or duplicate subcommands. In practice, it’s only `add` that make a distinctions between the two types.

Reply:
Ah, I wasn't privy to those conversations, for some reason I thought collections had their own tables and operations. In that case, your suggestion is much more elegant.